### ISSUE TITLE
Progress Bar on some windows 10 computers is too squished

### ISSUE THREAD

**Issue by sreilly** [TheInfiniteKind/moneydance_open#15]
On some Windows 10 computers, the small progress monitor window (need screenshot) is too small and compresses the progress bar so much that you can't see the progress.

**Comment by Derekkent23** [comment]
Import QIF file progress bar fixed in later Preview builds at some point after 2006.

**Comment by sreilly** [comment]
@Derekkent23 does this mean we can close this issue?

**Comment by Derekkent23** [comment]
Yes, you can close this issue. Not sure which preview build fixed this issue but when I imported a QIF file into 3065 as a test the progress bar worked correctly.
